src/components/LoginPage/Slide5.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import slideImage from 'components/LoginPage/images/slide-5.svg';
+import arrow from 'components/LoginPage/images/arrow.svg';
+
+const Slide = styled.section`
+  position: relative;
+   .shadow {
+    -webkit-filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+    filter: drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3));
+    }
+  .hint-old-notes {
+    position: absolute;
+    right: 5%;
+    width: 22%;
+    height: 11%;
+    top: 22%;
+    img {
+      width: 12%;
+      position: absolute;
+      bottom: 2%;
+      left: 44%;
+    }
+  }
+  .hint-recent-notes {
+    position: absolute;
+    left: 13%;
+    bottom: 3%;
+    width: 26%;
+    height: 13%;
+    img {
+      width: 12%;
+      position: absolute;
+      top: 2%;
+      left: 44%;
+    }
+  }
```

Comment:
The following code is duplicated for `.hint-old-notes` and `.hint-recent-notes`:
```
    position: absolute;
    img {
      width: 12%;
      position: absolute;
      bottom: 2%;
      left: 44%;
    }
```